config/webpack-config.js

```diff
@@ -36,3 +36,40 @@ module.exports = {
     new BannerFooterPlugin( '{% raw %}', '{% endraw %}', { raw: true } )
   ]
 };
+
+var ieConf = {
+  entry: paths.unprocessed + '/js/ie/common.ie.js',
+  output: {
+    filename: 'common.ie.js'
+  },
+  plugins: [
+    new webpack.optimize.UglifyJsPlugin( {
+      compress: { warnings: false }
+    } )
+  ]
+};
+
+var atomicConf = {
```

Comment:
All of our scripts should eventually be atomic, could this and others be updated to `onDemand` instead for clarity?